src/hooks/sdk/docs/useSelectVideoInputDevice.stories.mdx

```diff
@@ -51,3 +56,50 @@ const MyChild = () => {
 ### Dependencies
 
 - `AudioVideoProvider`
+
+
+### Why does my Webcam LED light turns on when I change a video input device using `selectVideo` from `useSelectVideoInputDevice` hook but no local video starts and the LED stays `ON`?
+
+Whenever you change and select a video input device using the `useSelectVideoInputDevice` hook, it internally calls the `chooseVideoInputDevice` API from the Amazon Chime SDK for JavaScript.
+The `chooseVideoInputDevice` API internally calls `getUserMedia` with requested video constraints.
+When successful, it triggers the LED light to go `ON`. Hence, we suggest to use the hook to select the video input device when you want to actually start the local video.
+
+Currently, the `useSelectVideoInputDevice` hook does not start the local video and hence the Webcam LED light remains `ON` but no video starts.
+To change this behavior would be a functionally breaking change and hence we provide below updated hook you can use to work-around this issue.
+The recommendation is to only select a video input device (Laptop or external camera) using Chime component library before you want to start the local video tile or preview video or if it has been already started.
+
+```js
+import { useMeetingManager } from '../../providers/MeetingProvider';
+import { useLocalVideo } from '../../providers/LocalVideoProvider';
+import { useAudioVideo } from '../../providers/AudioVideoProvider';
+
+export const useSelectVideoInputDevice = () => {
+  const meetingManager = useMeetingManager();
+  const audioVideo = useAudioVideo();
+  const { setIsVideoEnabled } = useLocalVideo();
+
+  const selectVideo = async (deviceId: string) => {
+    await meetingManager.selectVideoInputDevice(deviceId);
+    if (deviceId === 'none') {
+      // If you are doing this for video preview functionality, replace below call with audioVideo.stopVideoPreviewForVideoInput(videoElement);
+      audioVideo?.stopLocalVideoTile();
+      // This is must to keep the LocalVideoProvider with up-to-date state.
+      setIsVideoEnabled(false);
+    } else {
+      // If you are doing this for video preview functionality, replace below call with audioVideo.startVideoPreviewForVideoInput(videoElement);
+      audioVideo?.startLocalVideoTile();
+      // This is must to keep the LocalVideoProvider with up-to-date state.
+      setIsVideoEnabled(true);
+    }
+  }
+  return selectVideo;
+};
+```
+
+#### How to stop the LED light?
+- Call `selectVideo('none')`.
+- Call `meetingManager.leave()`.
+
+#### Are there any UI components affected with this?
+- `VideoInputControl` and `CameraSelection` components internally use and call the select video input device on device selection change. Hence, these will show the Webcam light `ON` when a video input changes and is a valid one providing a video stream.
```

Comment:
Do you mean `useSelectVideoInputDevice` when you say "call the select video input device" ?

Reply:
Have updated the documentation, so removed this.